### ISSUE TITLE
Vertex Wobble/Distortion in Vulkan Mode

### ISSUE THREAD

**Issue by OpenRift412** [BodbDearg/PsyDoom#29]
Could we get the option to enable the PSX-like vertex/texture wobbling in the Vulkan renderer? The Vulkan renderer is great, but I'd love to see some of those old quirks emulated there as well.

**Comment by BodbDearg** [comment]
Thank you for the suggestion! I've given this some thought and decided that it's beyond the scope of the Vulkan renderer.

In order to implement the artifacts and glitches mentioned you'd have to pretty much make the Vulkan renderer work the same way as the classic renderer, bringing with it all the limitations and imprecisions of old. The Vulkan renderer is designed to break free of these problems and provide a crisp display, so it kind of goes against the entire point of the new renderer.

There's also the problem that the artifacts would not look the same when scaled up to higher resolutions, meaning you would really have to limit yourself to the original PSX resolution for it to look 'right'. With that in mind there is not much point in making Vulkan behave more imprecisely, because you already have the classic renderer available to do a similar job at the original resolution.

So TL/DR if you want the original look along with all the glitches, the suggestion would be to use the classic renderer.
